hack/build-cross.sh

```diff
@@ -29,7 +29,8 @@ fi
 
 # Build the primary client/server for all platforms
 OS_BUILD_PLATFORMS=("${platforms[@]}")
-os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
+# Create a sub-shell so that we don't pollute the outer environment
+( os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}" )
```

Comment:
My question was, also, if we have a work-around for the `$GOBIN` thing for cross-compiles, why not use that work-around for everything and never leak a changed `$GOBIN`, thereby not needing the subshell at all?